wikipendium/wiki/models.py

```diff
@@ -135,6 +135,8 @@ def clean(self):
         if '/' in self.title:
             raise ValidationError('Title cannot contain slashes')
 
+        self.content.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
Yup, got that.
So this pull request didn't change anything then. New one incoming, I guess.